installation-manager/src/main/java/org/wildfly/core/instmgr/InstMgrHistoryHandler.java

```diff
@@ -38,6 +43,12 @@ public class InstMgrHistoryHandler extends InstMgrOperationStepHandler {
 
     @Override
     public void execute(OperationContext context, ModelNode operation) throws OperationFailedException {
+        System.out.println("!! InstMgrHistoryRuntimeHandler");
+        System.out.println(context.getCurrentStage());
+        PathAddress installerAddress = PathAddress.pathAddress(PathElement.pathElement(CORE_SERVICE, "installer"));
+        OperationEntry installerOpEntry = context.getRootResourceRegistration().getOperationEntry(installerAddress, "history-runtime");
+        context.addStep(installerOpEntry.getOperationHandler(), OperationContext.Stage.RUNTIME);
```

Comment:
We have to find a better approach that does not require splitting the history handle into "history-runtime" and "history", both handlers do the same, in addition, the InstallationReportHandler.java host version uses history and the server version the history-handler. I'll take a closed look to find a better approach.